userspace/falco/CMakeLists.txt

```diff
@@ -1,12 +1,12 @@
-include_directories(${PROJECT_SOURCE_DIR}/../sysdig/userspace/libsinsp/third-party/jsoncpp)
+include_directories("${PROJECT_SOURCE_DIR}/../sysdig/userspace/libsinsp/third-party/jsoncpp")
```

Comment:
I suggest to consider another coding style issue. Do you eventually care also for run time characteristcs of CMake commands?